src/Microsoft.ML.LightGBM/LightGbmMulticlassTrainer.cs

```diff
@@ -43,22 +41,22 @@ internal LightGbmMulticlassTrainer(IHostEnvironment env, Options options)
         /// Initializes a new instance of <see cref="LightGbmMulticlassTrainer"/>
         /// </summary>
         /// <param name="env">The private instance of <see cref="IHostEnvironment"/>.</param>
-        /// <param name="labelColumn">The name of The label column.</param>
-        /// <param name="featureColumn">The name of the feature column.</param>
-        /// <param name="weights">The name for the column containing the initial weight.</param>
-        /// <param name="numLeaves">The number of leaves to use.</param>
-        /// <param name="numBoostRound">Number of iterations.</param>
-        /// <param name="minDataPerLeaf">The minimal number of documents allowed in a leaf of the tree, out of the subsampled data.</param>
+        /// <param name="labelColumnName">The name of The label column.</param>
+        /// <param name="featureColumnName">The name of the feature column.</param>
+        /// <param name="exampleWeightColumnName">The name for the column containing the initial weight.</param>
```

Comment:
can you use this one:
`/// <param name="exampleWeightColumnName">The name of the example weight column (optional).</param>`

Reply:
On all your files.

---
In reply to: [262249292](https://github.com/dotnet/machinelearning/pull/2808#discussion_r262249292) [](ancestors = 262249292)

Reply:
I see we did the same in other places of the code. So we have a lot of optional parameters -- why are we marking only this one as (optional)?


---
In reply to: [262250845](https://github.com/dotnet/machinelearning/pull/2808#discussion_r262250845) [](ancestors = 262250845,262249292)

Reply:
Good question. I don't know. I'm just rooting for consistency.

---
In reply to: [262252375](https://github.com/dotnet/machinelearning/pull/2808#discussion_r262252375) [](ancestors = 262252375,262250845,262249292)